tests/resume.test.ts

```diff
@@ -220,3 +220,29 @@ describe('patch resume', () => {
     ).rejects.toThrowError(ForbiddenError);
   });
 });
+
+describe('delete resume', () => {
+  test('delete resume route successfully deletes user resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    await new PortalState().createUsers(member).write();
+
+    const resumeFile = FileFactory.pdf(Config.file.MAX_RESUME_FILE_SIZE / 2);
+    const fileLocation = 'fake location';
+
+    const storageService = Mocks.storage(fileLocation);
+    const resumeController = ControllerFactory.resume(
+      conn,
+      instance(storageService),
+    );
+    const request = { isResumeVisible: true };
+    await resumeController.uploadResume(resumeFile, request, member);
```

Comment:
Instead of relying on another controller call in a test that should only be testing `deleteResume`, let's use the `ResumeFactory` to generate a resume and use `PortalState::createResumes` to write that resume to the DB. That way, we can avoid this unnecessary `uploadResume()` call and go to directly testing `deleteResume()`.

See my comment on Meshach's PR here for more details/context: https://github.com/acmucsd/membership-portal/pull/317#discussion_r1093813060

Reply:
Yeah that was my initial thought process, but the process of desperately debugging led me astray. I can fix it real quick.